app/src/main/java/app/revanced/integrations/patches/LithoThemePatch.java

```diff
@@ -19,13 +17,22 @@ public class LithoThemePatch {
         -15790321, // comments chip background (new layout)
         -98492127 // video chapters list background
     };
+    private static final int solidBlack = -16777215;
+
+    private static final int trasparent = 0;
+
+    // boolean used to check if shorts comment box is currently visible
+    public static boolean shortsCommentBox = false;
```

Comment:
Usually getters and setters are used to access fields, but for that a seperate PR should be opened to fix this across the entire project consistently.